stock_critical_forecast_link_replenish/tests/TEST_INSTRUCTIONS.rst

```diff
@@ -0,0 +1,8 @@
+- Open the Inventory app
+- Create the replenishment rule for a product, set the minimum and maximum quantities
+- Set the route to "Buy", assign a vendor
+- Under Operations menu click "Calculate Critical Forecast"
+- Under Reporting menu open the Critical Forecast
+- Critical forecast is shown for products in inventory, manufacturing and reordering
```

Comment:
This is not correct. The critical forecast shows products with a critical order date.